### PR TITLE
Init copier template

### PR DESCRIPTION
**Pull request by joao-p-marques**
This bootstraps this repo with the https://github.com/OCA/oca-addons-repo-template/ to update pre-commit (ping @pedrobaeza https://github.com/OCA/timesheet/pull/420#issuecomment-754048977) and help keep it up to date.

### CODE REVIEW

File: .isort.cfg [hunk 1]
```diff
@@ -10,4 +10,3 @@ known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
 default_section=THIRDPARTY
-known_third_party=babel,dateutil,openupgradelib
```
Comment (joao-p-marques) [inline .isort.cfg:13]: Copier deleted this line according to the template, but I am not sure if this should be added back.

Comment (pedrobaeza) [reply .isort.cfg:13]: It's not needed.

**Comment by pedrobaeza** [comment]
Please fix Travis

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -72,18 +63,30 @@ repos:
...
+        exclude: /__init__\.py$
+        additional_dependencies: ["flake8-bugbear==19.8.0"]
+      - id: flake8
+        name: flake8 only __init__.py
+        args: ["--extend-ignore=F401"] # ignore unused imports in __init__.py
+        files: /__init__\.py$
+        additional_dependencies: ["flake8-bugbear==19.8.0"]
   - repo: https://github.com/pre-commit/mirrors-pylint
-    rev: v2.3.1
+    rev: v2.5.3
```
Comment (pedrobaeza) [inline .pre-commit-config.yaml:79]: Are you sure these changes are correct, or they are v14 template?

Comment (joao-p-marques) [reply .pre-commit-config.yaml:79]: Yes, this comes from the upstream template: https://github.com/OCA/oca-addons-repo-template/blob/master/version-specific/13.0/.pre-commit-config.yaml#L113

Comment (pedrobaeza) [reply .pre-commit-config.yaml:79]: OK, it's curious that this one is so outdated. Then you will need to fix the appearing errors due to this change.

Comment (joao-p-marques) [reply .pre-commit-config.yaml:79]: Ok. However, I don't get any pre-commit errors when running `pre-commit run -a` locally.

**Comment by joao-p-marques** [comment]
Travis seems to be failing due to `web_widget_x2many_2d_matrix` being missing, but I didn't change anything in the travis config that could produce that error. Could you relaunch the build to see if that was a temporary error?

File: oca_dependencies.txt [hunk 3]
```diff
@@ -1,4 +1 @@
-# list the OCA project dependencies, one per line
-# add a github url if you need a forked version
-project
```
Comment (pedrobaeza) [inline oca_dependencies.txt:3]: This is why it's failing. You shouldn't remove these lines.

Comment (joao-p-marques) [reply oca_dependencies.txt:3]: Thank you! Change made.

**Pull request merged**